pkgs/games/betterspades/default.nix

```diff
@@ -0,0 +1,286 @@
+        sha256 = ver.log.c.sha256;
+      };
+
+      "hashtable.c" = fetchurl {
+        url = "https://raw.githubusercontent.com/goldsborough/hashtable/${ver.hashtable.commit}/hashtable.c";
+        sha256 = ver.hashtable.c.sha256;
+      };
+      "hashtable.h" = fetchurl {
+        url = "https://raw.githubusercontent.com/goldsborough/hashtable/${ver.hashtable.commit}/hashtable.h";
+        sha256 = ver.hashtable.h.sha256;
+      };
+
+      "libdeflate.h" = fetchurl {
+        url = "https://raw.githubusercontent.com/ebiggers/libdeflate/${ver.libdeflate.commit}/libdeflate.h";
+        sha256 = ver.libdeflate.sha256;
+      };
+
+      "microui.c" = fetchurl {
+        url = "https://raw.githubusercontent.com/rxi/microui/${ver.microui.commit}/src/microui.c";
+        sha256 = ver.microui.c.sha256;
+      };
+      "microui.h" = fetchurl {
+        url = "https://raw.githubusercontent.com/rxi/microui/${ver.microui.commit}/src/microui.h";
+        sha256 = ver.microui.h.sha256;
+      };
+    };
+in
+stdenv.mkDerivation rec {
+  pname = "betterspades";
+  version = betterspades-v.version;
+
+  src = srcs.betterspades;
+  icon = "${src}/resources/icon.png";
+
+  nativeBuildInputs = [ cmake pkg-config copyDesktopItems dos2unix ];
+
+  buildInputs = [
+    glfw3
+    glew
+    enet
+    cglm
+    openal
+    libdeflate
+    libX11
+    libXau
+    libXdmcp
+    libSM
+    libXext
+  ];
+
+  cmakeFlags = [
+    "-DFETCHCONTENT_SOURCE_DIR_VXL=${srcs.libvxl}"
+    "-DFETCHCONTENT_SOURCE_DIR_CGLM=${cglm.src}"
+  ];
+
+  # all .c and .h files in repo have CRLF line endings. This makes patching difficult.
+  prePatch = ''
+    find . -name '*.c' -exec dos2unix {} +
+    find . -name '*.h' -exec dos2unix {} +
+  '';
+
+  patches = [
+    ./001-dont-install-OpenAL32.dll.patch
+    ./002-fix-macos-build.patch
+  ];
+
+  postUnpack = ''
+    cp ${srcs."ini.c"} source/src/ini.c
+    cp ${srcs."ini.h"} source/src/ini.h
+
+    mkdir -p source/src/lodepng
+    cp ${srcs."lodepng.cpp"} source/src/lodepng/lodepng.c
+    cp ${srcs."lodepng.h"} source/src/lodepng/lodepng.h
+
+    cp ${srcs."dr_wav.c"} source/src/dr_wav.c
+
+    cp ${srcs."stb_truetype.h"} source/src/stb_truetype.h
+
+    cp ${srcs."parson.c"} source/src/parson.c
+    cp ${srcs."parson.h"} source/src/parson.h
+
+    cp ${srcs."http.h"} source/src/http.h
+
+    cp ${srcs."log.c"} source/src/log.c
+    cp ${srcs."log.h"} source/src/log.h
+
+    cp ${srcs."hashtable.c"} source/src/hashtable.c
+    cp ${srcs."hashtable.h"} source/src/hashtable.h
+
+    cp ${srcs."libdeflate.h"} source/deps/libdeflate.h
+
+    cp ${bsresources} source/bsresources.zip
+
+    cp ${srcs."microui.c"} source/src/microui.c
+    cp ${srcs."microui.h"} source/src/microui.h
+  '';
+
+  postInstall = ''
+      cd $out
+      rm -rf -- cache/ include/ lib/ logs/ screenshots/ config.ini
+      mkdir -p -- bin
+      mv -- client bin/.betterspades-wrapped
```

Comment:
.*-wrapped binaries are usually created with makeWrapper and not manually created